### PR TITLE
[WIP] Russian Language Support

### PR DESCRIPTION
**Pull request by Nekonyx**
Phrases translated: 298 of 975 [30%]

Files:
[100%] [140/140] layout.php
[89%] [89/99] users.php
[100%] [31/31] common.php
[100%] [20/20] home.php
[91%] [11/12] events.php
[100%] [7/7] wiki.php
[0%] [0/2] validation.php
[0%] [0/16] user_verification.php
[0%] [0/12] store.php
[0%] [0/21] status_page.php
[0%] [0/13] ranking.php
[0%] [0/19] password_reset.php
[0%] [0/12] news.php
[0%] [0/33] multiplayer.php
[0%] [0/21] model_validation.php
[0%] [0/3] livestreams.php
[0%] [0/31] accounts.php
[0%] [0/19] admin.php
[0%] [0/11] errors.php
[0%] [0/25] contest.php
[0%] [0/42] community.php
[0%] [0/6] changelog.php
[0%] [0/58] beatmapsets.php
[0%] [0/124] beatmaps.php
[0%] [0/5] beatmap_discussions.php
[0%] [0/14] bbcode.php
[0%] [0/49] authorization.php
[0%] [0/11] artist.php
[0%] [0/1] api.php
[0%] [0/118] forum.php

Demo: nneko.ru

### CODE REVIEW

File: resources/lang/ru/home.php [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+return [
+    'landing' => [
+        'online' => 'из них <strong>:players</strong> сейчас в <strong>:games</strong> играх',
+        'peak' => 'Пик :count активных игроков',
+        'players' => '<strong>:count</strong> зарегистрированных игроков',
+
+        'download' => [
+            '_' => 'Скачать игру',
+            'soon' => 'osu! пока доступен только для Windows',
```
Comment (stanriders) [inline resources/lang/ru/home.php]: "доступна"

File: resources/lang/ru/layout.php [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+            '_' => 'карты',
+            'show' => 'информация',
+            'index' => 'список',
+            'artists' => 'избранные исполнители',
+            // 'getPacks' => 'packs',
+            // 'getCharts' => 'charts',
+        ],
+        'beatmapsets' => [
+            '_' => 'карты',
+            'discussion' => 'моддинг',
```
Comment (stanriders) [inline resources/lang/ru/layout.php]: "обсуждения"?

File: resources/lang/ru/layout.php [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+        ],
+        'ranking' => [
+            '_' => 'ранкинг',
+            'getOverall' => 'общий',
+            'getCountry' => 'регионы',
```
Comment (stanriders) [inline resources/lang/ru/layout.php]: "страны"

File: resources/lang/ru/layout.php [hunk 4]
```diff
@@ -0,0 +1,243 @@
...
+            'getMapper' => 'мапперы',
+            'index' => 'общий',
+        ],
+        'community' => [
+            '_' => 'сообщество',
+            'dev' => 'osu!dev',
+            'getForum' => 'форум',
+            'getChat' => 'чат',
+            'getSupport' => 'поддержка',
+            'getLive' => 'стримы по osu!',
```
Comment (stanriders) [inline resources/lang/ru/layout.php]: "трансляции" уж скорее, так как это более литературное слово

File: resources/lang/ru/layout.php [hunk 5]
```diff
@@ -0,0 +1,243 @@
...
+            'forum-forums-index' => 'форум',
+            'forum-forums-show' => 'форум',
+        ],
+        'multiplayer' => [
+            '_' => 'мультиплеер',
+            'show' => 'матчи',
+        ],
+        'error' => [
+            '_' => 'ошибка',
+            '404' => 'потеряно',
```
Comment (stanriders) [inline resources/lang/ru/layout.php]: "не найдено"? Или в оригинале "lost"?

File: resources/lang/ru/layout.php [hunk 6]
```diff
@@ -0,0 +1,243 @@
...
+            'logout' => 'Выход',
+            'help' => 'Помощь',
+        ],
+        'store' => [
+            '_' => 'магазин',
+            'getListing' => 'товары',
+            'getCart' => 'корзина',
+
+            'getCheckout' => 'проверка',
+            'getInvoice' => 'квитанция',
```
Comment (stanriders) [inline resources/lang/ru/layout.php]: Как мне кажется, больше бы подошло слово "чек"

File: resources/lang/ru/layout.php [hunk 7]
```diff
@@ -0,0 +1,243 @@
...
+                'show' => 'детали',
+            ],
+        ],
+    ],
+
+    'footer' => [
+        'general' => [
+            '_' => 'Общее',
+            'home' => 'Главная',
+            'changelog' => 'Список изменении',
```
Comment (stanriders) [inline resources/lang/ru/layout.php]: "Список изменени**й**"

File: resources/lang/ru/layout.php [hunk 8]
```diff
@@ -0,0 +1,243 @@
...
+            'link' => false,
+        ],
+        '403' => [
+            'error' => 'Вы не должны быть здесь.',
+            'description' => 'Вы можете вернуться назад.',
+            'link' => false,
+        ],
+        '401' => [
+            'error' => 'Вы не должны быть здесь.',
+            'description' => 'Вы можете вернуться назад. Или, может авторизироваться.',
```
Comment (stanriders) [inline resources/lang/ru/layout.php]: "Или, возможно, авторизоваться."

Comment (Nekonyx) [inline resources/lang/ru/layout.php]: Спасибо за правки. Внесу их сразу же, как появится возможность).

**Comment by kj415j45** [comment]
some section have been changed, please check them carefully

**Pull request closed without merging**